readme.md

```diff
@@ -76,6 +76,19 @@ then to run the tests:
 yarn test
 ```
 
+In order to generate benchmarks locally during development:
+
+```bash
+yarn testnet-b
```

Comment:
Can we use `testnet:benchmark` instead? All the other scripts use this convention

Reply:
Thanks, Implemented 👍